samples/gatsby-preview/src/index.js

```diff
@@ -8,21 +8,6 @@ import './index.css';
 class App extends React.Component {
   constructor(props) {
     super(props);
-
-    const { parameters } = this.props.sdk;
-    const { webhookUrl, previewUrl } = parameters.installation;
-    const { contentTypeSlug } = parameters.instance;
-
-    let slug = contentTypeSlug ? contentTypeSlug : '';
-
-    if (this.props.sdk.entry.fields.slug) {
-      slug += '/' + this.props.sdk.entry.fields.slug.getValue();
-    }
-
-    this.state = {
```

Comment:
None of this is really necessary, we can (and should!) just derive from props when we can.